libfwupdplugin/meson.build

```diff
@@ -236,7 +236,7 @@ library_deps = [
   platform_deps,
 ]
 
-if get_option('rust').allowed()
+if rustc.found()
```

Comment:
shouldn't this be something like `if get_option('rust').disable_auto_if(not rustc.found()).allowed()` ?

Reply:
Yeah I think you're right